Modules/SubModule_Import/xASL_imp_PreallocateGlobalCounts.m

```diff
@@ -0,0 +1,40 @@
+function [subject,visit] = xASL_imp_PreallocateGlobalCounts(nSubjects,subject,visit)
```

Comment:
Do we use this multiple times, in different functions? Otherwise it's nicer to keep this as a subfunction instead.

Reply:
Not sure if I agree. In which case is a long single script with a ton of sub-functions better than a higher number of smaller scripts? I know it could be easier to understand which sub-function are for which module/sub-module, but on the other hand it's way easier to work on larger repositories if you have a higher number of smaller scripts. Easier to maintain IMHO.